compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -372,11 +372,22 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
 
 void QuantizeWithMinMaxPass::set_input_type(loco::Graph *g) const
 {
+  assert(g); // FIX_CALLER_UNLESS
   auto inputs = g->inputs();
-  for (auto node : loco::input_nodes(g))
+
+  assert(inputs);                                    // FIX_CALLER_UNLESS
+  assert(inputs->size() == _ctx->input_type.size()); // FIX_CALLER_UNLESS
+
+  // NOTE loco::input_nodes returns input nodes following the order of InputIndex
+  auto input_nodes = loco::input_nodes(g);
+  for (uint32_t i = 0; i < input_nodes.size(); i++)
```

Comment:
Changed from `for (auto node : loco::input_nodes(g))` to access index `i`. `i` is used to get `_ctx->input_type[i]`.